pkg/webui/console/store/actions/lib.js

```diff
@@ -0,0 +1,34 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* eslint-disable import/prefer-default-export */
+
+const createRequestActions = function (baseType) {
+  const requestType = `${baseType}_REQUEST`
+  const successType = `${baseType}_SUCCESS`
+  const failureType = `${baseType}_FAILURE`
+
+  return [{
+    request: requestType,
+    success: successType,
+    failure: failureType,
+  },
+  {
+    request: payload => ({ type: requestType, payload }),
```

Comment:
please check https://redux-actions.js.org/api/createaction and either copy their api or include the lib. 
`getApplication({ id: appId })` - is not the right way to go